wrappers/services.go

```diff
@@ -349,7 +376,7 @@ type AddSnapServicesOptions struct {
 }
 
 // AddSnapServices adds service units for the applications from the snap which are services.
```

Comment:
I think the doc-comment should clarify that it does not enable or start services